internal/plugin/client.go

```diff
@@ -128,6 +128,7 @@ func (c *Client) GetByName(ctx context.Context, name string) (*Plugin, error) {
 
 	query := baseURL.Query()
 	query.Set("name", name)
+	query.Set("hidden", "true")
```

Comment:
@mmatur @juliens according to the review comments on https://github.com/traefik/plugin-service/pull/56 this should `hidden` should be false.
```suggestion
	query.Set("hidden", "false")
```